targets/lenovo-x1/appvms/gala.nix

```diff
@@ -10,6 +15,9 @@
   extraModules = [
     {
       time.timeZone = "Asia/Dubai";
```

Comment:
https://github.com/vilvo/ghaf/commit/dd213b5689d4e763cb019d9b7cf422e9c873a138

Reply:
This commit applied in [7af8203](https://github.com/tiiuae/ghaf/commit/7af8203a2f38b76e30b15636ec6f19ce9f62c4a9).